doc/articles/get-started-next-steps.md

```diff
@@ -19,9 +19,9 @@ Visit [the documentation](external/uno.extensions/doc/Overview/ExtensionsOvervie
 
 Syncfusion supports Uno Platform to two of its most popular UWP controls —DataGrid and Charts, published [Uno Platform Succinctly]( https://www.syncfusion.com/succinctly-free-ebooks/uno-platform-succinctly). A free eBook that will take you from setting up your first Uno project in Visual Studio to a fully functioning single-page application.
 
-#### Andrew Hoefling – Getting Started with Uno Platform
```

Comment:
ℹ️ This is miscellaneous but I wanted to include it for Skye's sake, since I know her. Skye transitioned from Andrew and her proper name is reflected elsewhere in these docs; looks like it just got missed here.